index.html

```diff
@@ -51,9 +51,60 @@
     <script src="compiler.js"></script>
     <script src="try-slang.js"></script>
     <script src="slang-wasm.js"></script>
+    <script src="language-server.js"></script>
+    <style>
+      #loading-screen {
```

Comment:
should we put the style code to css file?

Reply:
It was originally in the css file, but for some reason my browser couldn't pick them up.
It is likely due to stale browser cache, and we should be able to move them back.

Reply:
yes, it must be the cache thing.
I sometimes have to delete the browser cache after I change the code.